tests/src/test/scala/cats/tests/CokleisliTests.scala

```diff
@@ -19,18 +19,4 @@ class CokleisliTests extends CatsSuite {
 
   checkAll("Cokleisli[Option, Int, Int]", ProfunctorTests[Cokleisli[Option, ?, ?]].profunctor[Int, Int, Int, Int, Int, Int])
   checkAll("Profunctor[Cokleisli[Option, ?, ?]", SerializableTests.serializable(Profunctor[Cokleisli[Option, ?, ?]]))
-
-  {
-    // Ceremony to help scalac to do the right thing, see also #267.
-    type CokleisliNEL[A, B] = Cokleisli[NonEmptyList, A, B]
-
-    implicit def ev0[A: Arbitrary, B: Arbitrary]: Arbitrary[CokleisliNEL[A, B]] =
-      cokleisliArbitrary
-
-    implicit def ev1[A: Arbitrary, B: Eq]: Eq[CokleisliNEL[A, B]] =
-      cokleisliEq[NonEmptyList, A, B](oneAndArbitrary, Eq[B])
-
-    checkAll("Cokleisli[NonEmptyList, Int, Int]", ArrowTests[CokleisliNEL].arrow[Int, Int, Int, Int, Int, Int])
-    checkAll("Arrow[Cokleisli[NonEmptyList, ?, ?]]", SerializableTests.serializable(Arrow[CokleisliNEL]))
```

Comment:
These are no longer valid.

We need an `Arrow[CokleisliNEL]`.
This is equivalent to wanting `Arrow[Cokleisli[NonEmptyList, ?, ?]]`,
e.g. `Arrow[Cokleisli[OneAnd[?, List], ?, ?]]`.

The implicit that would be used here is:

```scala
  implicit def cokleisliArrow[F[_]](implicit ev: Comonad[F]): Arrow[Cokleisli[F, ?, ?]] =
    new CokleisliArrow[F] { def F: Comonad[F] = ev }
```

This requires a `Comonad[F]`, e.g. a `Comonad[OneAnd[?, List]]`.

However, the previous comonad instance for `OneAnd` was invalid, so I removed it.

Maybe I should implement an instance specifically for `NonEmptyList`?